contrib/babelfishpg_unit/README.md

```diff
@@ -0,0 +1,66 @@
+# How to add unit tests for babelfish?
+
+Babelfish has introduced a new extension named babelfishpg_unit which enables us to run unit tests. Please follow the README.md file present in contrib/README.md. Once we have built and installed the extension, we are ready to write unit tests. 
+
+* To maintain a well-organized structure, the directory layout will be as follows:
+
+- babelfish_extensions
+    -- contrib
+        --- babelfishpg_common
+        --- babelfishpg_money
+        --- babelfishpg_tsql
+        --- babelfishpg_tds
+        --- babelfishpg_unit
+            ---- Makefile
+            ---- babelfishpg_unit.control
+            ---- babelfishpg_unit--1.0.0.sql 
+            ---- babelfishpg_unit.c 
+            ---- babelfishpg_unit.h 
+            ---- test_1.c
+            ---- test_2.c
+
+As we can see, test_1.c, test_2.c, ... are the individual test files which contains the test cases to a particular area of functionality or component being tested.
```

Comment:
Here, test_1.c, test_2.c, ... are the individual test files containing the test cases.

No need to specify how they are structured as you already are mentioning it in the how to add test section.

Reply:
Done!